.github/workflows/lint-check.yml

```
@@ -0,0 +1,102 @@
+        with:
+          version: '10'
+          
+      - name: 📥 Install dependencies
+        run: pnpm install --frozen-lockfile
+        
+      - name: 🧪 ESLint Check
+        id: eslint
+        run: |
+          echo "## 🧪 ESLint Report" >> $GITHUB_STEP_SUMMARY
+          pnpm run lint 2>&1 | tee lint-output.txt || true
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)